src/core/checkedint.d

```diff
@@ -517,3 +519,20 @@ unittest
     assert(mulu(0UL, 0UL, overflow) == 0);
     assert(overflow);                   // sticky
 }
+
+
+private:
+
+pragma(inline, true)
+@property size_t _signBit(T)(in T n)
```

Comment:
You know it, private symbols can cause name collisions when importing a module.
See https://github.com/D-Programming-Language/dmd/pull/739 or [DIP22](http://wiki.dlang.org/DIP22) for more info.